dart/lib/src/version.dart

```diff
@@ -8,19 +8,28 @@
 /// This library contains Sentry.io SDK constants used by this package.
 library version;
 
+import 'utils.dart';
+
 /// The SDK version reported to Sentry.io in the submitted events.
 const String sdkVersion = '4.0.0';
 
+String get sdkName => isWeb ? browserSdkName : ioSdkName;
```

Comment:
we can make those const private now, if what matters is the `sdkName`

Reply:
✅